src/Snowcap/Emarsys/Client.php

```diff
@@ -112,12 +112,15 @@ public function __construct(
     /**
      * @param array $mapping
      */
-    public function addFieldsMapping($mapping = []): void
+    public function addFieldsMapping(array $mapping = []): void
     {
         $this->fieldsMapping = array_merge($this->fieldsMapping, $mapping);
     }
 
-    public function addChoicesMapping($mapping = []): void
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
I'm never 100% sure, but I didn't think these were really necessary if you're not changing or adding anything (i.e. I thought it just picks up the inherited docs by default?)